From: John Wright <jsw@debian.org>
Date: Thu, 29 Dec 2011 21:38:32 -0700
Subject: Dynamically link makedumpfile

Debian does not have a static version of libebl, so it's not possible to
build it statically.  Additionally, the symbols normally in the libebl
dynamic library are merged into libdw in Debian, so we also drop the -lebl
linker flag.

Signed-off-by: John Wright <jsw@debian.org>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index d643d2b..f7373d1 100644
--- a/Makefile
+++ b/Makefile
@@ -38,7 +38,7 @@ $(OBJ_ARCH): $(SRC_ARCH)
 	$(CC) $(CFLAGS_ARCH) -c -o ./$@ ./$(@:.o=.c) 
 
 makedumpfile: $(SRC) $(OBJ_PART) $(OBJ_ARCH)
-	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJ_PART) $(OBJ_ARCH) -o $@ $< -static -ldw -lbz2 -lebl -ldl -lelf -lz
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJ_PART) $(OBJ_ARCH) -o $@ $< -ldw -lbz2 -ldl -lelf -lz
 	echo .TH MAKEDUMPFILE 8 \"$(DATE)\" \"makedumpfile v$(VERSION)\" \"Linux System Administrator\'s Manual\" > temp.8
 	grep -v "^.TH MAKEDUMPFILE 8" makedumpfile.8 >> temp.8
 	mv temp.8 makedumpfile.8
-- 
